clamav-notify/lighttpd.conf

```
@@ -0,0 +1,9 @@
+server.document-root = "/var/lib/clamav/" 
+
+server.port = 8080
+
+mimetype.assign = (
+  ".htm" => "text/html", 
```

Don't know much about `lighttpd` but this doesn't look right
```suggestion
  ".html" => "text/html", 
```

---

Just preference/costmetic, this makes no difference